components/browser/state/src/main/java/mozilla/components/browser/state/reducer/InProgressMediaReducer.kt

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.reducer
+
+import mozilla.components.browser.state.action.InProgressMediaAction
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.state.TabSessionState
+
+/**
+ * [InProgressMediaAction] reducer that will update [BrowserState].
+ */
+object InProgressMediaReducer {
```

Comment:
With the suggestion above we wouldn't need another reducer. The `LastAccessReducer` could take care of it.